portfolio/src/main/java/com/google/sps/data/CalendarEvent.java

```diff
@@ -47,4 +47,12 @@ public Instant getStartTimeInstant() {
   public Instant getEndTimeInstant() {
     return endTime;
   }
+
+  public long getStartTimeLong() {
```

Comment:
Here and below, I don't think the `Long` suffix is useful here. What would be more useful is if the suffix was the units instead, like `getStartTimeSeconds()`.

Reply:
I removed the methods.